new-frontiers-14/frontier-station-14
# Replace Tinnia's Rest POI tiny fans with directional fans

## About the PR
Replaced the tiny fans on the Tinnia's Rest POI with directional fans (yes, even those pesky ones in the freezer).

## Why / Balance
Compliance.

## How to test
Load in the grid and have  walk around, see that tiny fans have been purged from the land.

## Media
![dock1](https://github.com/user-attachments/assets/2b3f7dd9-14e5-45d7-826b-72d2614b86e0)
![dock2](https://github.com/user-attachments/assets/bd127bde-15db-43fa-9d0c-35c482c1539e)
![dock3](https://github.com/user-attachments/assets/10c368f1-0e74-4cc1-a6f1-12732292a2a4)
![freezer](https://github.com/user-attachments/assets/8ebc271c-f094-49c3-8496-6a25a85d5e85)
![solars](https://github.com/user-attachments/assets/d43350db-ac15-4c4b-aef1-b1271bf40253)

## Requirements



## Breaking changes
None

**Changelog**

:cl:
- tweak: Replaced tiny fans with directional fans on Tinnia's Rest POI
